VideoIndexerEnabledByArc/single-node-k8s-cluster/single-node.k8s.bicep

```diff
@@ -0,0 +1,215 @@
+@description('Azure Resource Name Prefux')
+param prefix string
+
+@description('Specifies control plane node VM size.')
+param controlPlaneNodeVmSize string = 'Standard_DS32a_v4'
+
+@description('Specifies the location.')
+param location string = resourceGroup().location
+
+@description('Specifies the address prefixes of the virtual network.')
+param virtualNetworkAddressPrefixes string = '10.0.0.0/16'
+
+@description('Specifies the address prefix of the subnet which contains the virtual machine.')
+param vmSubnetAddressPrefix string = '10.0.1.0/24'
+
+@description('Specifies the name of the network security group associated to the subnet hosting the virtual machine.')
+param vmSubnetNsgName string = 'VmSubnetNsg'
+
+var vnetName = 'Vm${prefix}vnet'
+var subnetName = 'Vm${vnetName}-subnet0'
+var nicName = 'Vm${prefix}nic'
+
+var vmName = 'k8s-master'
```

Comment:
use the prefix in the vm's name

Reply:
this is by-design . as I had issues with the local-path provisioner which did not work on any node name with ENV Variable 
see local-path-provisioner.yaml line 131